src/PointwiseFunctions/MathFunctions/Gaussian.hpp

```diff
@@ -39,9 +39,20 @@ class Gaussian : public MathFunction<1> {
   static constexpr OptionString_t help = {
       "Applies a Gaussian function to the input value"};
 
+  WRAPPED_PUPable_decl_template(Gaussian);  // NOLINT
+
+  explicit Gaussian(CkMigrateMessage* /*unused*/) {}
```

Comment:
We can noexcept all the migration constructors, Charm++ does no exception handling.